ibmdbpy/base.py

```diff
@@ -1177,16 +1177,11 @@ def add_column_id(self, idadf, column_id="ID", destructive=False):
 
         Notes
         -----
-            The non-destructive creation of column IDs is not reliable, because 
-            row IDs are recalculated on the fly in a non-deterministic way. The 
-            only reliable way is to create it destructively, but the row IDs 
-            will be created at random. This could be highly improved in the 
-            future. An idea is to create ID columns in a non-destructive way 
-            and base them on the sorting of a set of columns, defined by the 
-            user, or all columns if no column combination results in unique 
-            identifiers.
-            
-            This method is DEPRECATED. 
+            The non-destructive creation of column IDs is not reliable, because row IDs are recalculated on the fly in a non-deterministic 
+            way each time a new view is produced. On the contrary, creating them destructively id est physically is reliable but can take time. 
```

Comment:
```suggestion
            way each time a new view is produced. On the contrary, creating them destructively (i.e. physically) is reliable but can take time. 
```

(I've never seen i.e. spelled out before, so take this suggestion with a huge grain of salt)

Reply:
Well :') I've changed it... It's true that people tend to forget that i.e comes from latin so they are surprised when it is fully spelled.